cpp/src/centrality/README.md

```diff
@@ -0,0 +1,64 @@
+# Centrality algorithms
+cuGraph Pagerank is implemented using our graph primitive library
+
+## Pagerank
+
+The unit test code is the best place to search for examples on calling louvain.
+
+ * [SG Implementation](../../tests/experimental/pagerank_test.cpp)
+ * [MG Implementation](../../tests/pagerank/mg_pagerank_test.cpp)
+
+## Simple pagerank
+
+The example assumes that you create an SG or MG graph somehow.  The caller must create the pageranks vector in device memory and pass in the raw pointer to that vector into the pagerank function.
+
+```cpp
+#include <algorithm.hpp>
+...
+using vertex_t = int32_t;       // or int64_t, whichever is appropriate
+using weight_t = float;         // or double, whichever is appropriate
+using result_t = weight_t;      // could specify float or double also
+raft::handle_t handle;          // Must be configured if MG
+auto graph_view = graph.view(); // assumes you have created a graph somehow
+
+result_t constexpr alpha{0.85};
+result_t constexpr epsilon{1e-6};
+
+rmm::device_uvector<result_t> pageranks_v(graph_view.get_number_of_vertices(), handle.get_stream());
+
+// pagerank optionally supports three additional parameters:
+//     max_iterations - maximum number of iterations, if pagerank doesn't coverge by then we abort
+//     has_initial_guess - if true, values in the pagerank array when the call is initiated will be used as the initial pagerank values
+//     do_expensive_check - perform extensive validation of the input data before executing algorithm
+cugraph::experimental::pagerank(handle, graph_view, nullptr, nullptr, nullptr, vertex_t{0}, pageranks_v.data(), alpha, epsilon);
```

Comment:
No, I guess not. I can find the doc elsewhere so it was more a matter of convenience for the reader. It might be better to not include the redundant doc here then if you think it's likely to go stale.